packages/waku/src/lib/handlers/dev-worker-impl.ts

```diff
@@ -142,16 +128,21 @@ const loadEntries = async (config: ResolvedConfig) => {
 parentPort!.on('message', (mesg: MessageReq) => {
   if (mesg.type === 'render') {
     handleRender(mesg);
-  } else if (mesg.type === 'buf') {
-    const controller = controllerMap.get(mesg.id)!;
-    controller.enqueue(new Uint8Array(mesg.buf, mesg.offset, mesg.len));
-  } else if (mesg.type === 'end') {
-    const controller = controllerMap.get(mesg.id)!;
-    controller.close();
-  } else if (mesg.type === 'err') {
+  } else if (mesg.type === 'pipe') {
     const controller = controllerMap.get(mesg.id)!;
-    const err =
-      mesg.err instanceof Error ? mesg.err : new Error(String(mesg.err));
-    controller.error(err);
+    mesg.stream.pipeTo(
```

Comment:
I don't think this is necessary.
So, we don't need mesg.type=pipe at all, and attach the `stream` for mesg.type=render.